starter-code/books-coffee/views/new.ejs

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+  </head>
+  <body>
+
+    <h2>Add a Place</h2>
+
+    <form action="/new" method="post">
+      <input type="text" name="name" placeholder="name">
+      <input type="text" name="type" placeholder="type">
```

Comment:
In your Mongoose model you used an enum. However here the user can type anything. A `select` or a pair of radio buttons would be much more appropriate.

With the current code, if I type something unexpected, not only will it fail but it will fail *silently*. By that I mean that no error is displayed to the user; the browser just waits for an answer that never comes.